commands/init.ts

```diff
@@ -73,95 +76,125 @@ const personalAccessKeyConfigCreationFlow = async (env, account) => {
     logError(e);
   }
   return updatedConfig;
-};
+}
 
-const oauthConfigCreationFlow = async env => {
-  const configData = await promptUser(OAUTH_FLOW);
-  const accountConfig = {
+async function oauthConfigCreationFlow(env: Environment) {
+  const configData = await promptUser<OauthPromptResponse>(OAUTH_FLOW);
+  const accountConfig: OAuth2ManagerAccountConfig = {
     ...configData,
     env,
   };
   await authenticateWithOauth(accountConfig);
-  updateDefaultAccount(accountConfig.name);
+  updateDefaultAccount(accountConfig.name!);
   return accountConfig;
-};
-
-const CONFIG_CREATION_FLOWS = {
-  [PERSONAL_ACCESS_KEY_AUTH_METHOD.value]: personalAccessKeyConfigCreationFlow,
-  [OAUTH_AUTH_METHOD.value]: oauthConfigCreationFlow,
-};
+}
 
 const AUTH_TYPE_NAMES = {
   [PERSONAL_ACCESS_KEY_AUTH_METHOD.value]: PERSONAL_ACCESS_KEY_AUTH_METHOD.name,
   [OAUTH_AUTH_METHOD.value]: OAUTH_AUTH_METHOD.name,
 };
 
-exports.command = 'init';
-exports.describe = i18n(`${i18nKey}.describe`, {
+export const command = 'init';
+export const describe = i18n(`${i18nKey}.describe`, {
   configName: DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME,
 });
 
-exports.handler = async options => {
+type InitArgs = CommonArgs &
+  ConfigArgs &
+  TestingArgs & {
+    authType?: string;
+    account?: string;
```

Comment:
Can use `AccountArgs` here too

Reply:
Done in [be71b49](https://github.com/HubSpot/hubspot-cli/pull/1401/commits/be71b495f9f24049001fda5d7c3328c7479fcdc8)